pandas/tests/extension/base/ops.py

```diff
@@ -61,10 +62,10 @@ class BaseArithmeticOpsTests(BaseOpsUtil):
     * divmod_exc = TypeError
     """
 
-    series_scalar_exc = TypeError
-    frame_scalar_exc = TypeError
-    series_array_exc = TypeError
-    divmod_exc = TypeError
+    series_scalar_exc = TypeError  # type: Union[Type[Exception], None]
```

Comment:
```suggestion
    series_scalar_exc = TypeError  # type: Optional[Type[TypeError]]
```

I _think_ we don't need to be more permissive on the error type here.

also, we tend to use `Optional[<type>]` instead of `Union[<type>, None]`